src/Calendar.xml

```diff
@@ -0,0 +1,389 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<widget id="com.mendix.widget.custom.calendar.Calendar" needsEntityContext="true" offlineCapable="true" xmlns="http://www.mendix.com/widget/1.0/">
+    <name>Calendar</name>
+    <description>Display and manage calendar events</description>
+    <icon>iVBORw0KGgoAAAANSUhEUgAAAEAAAABACAYAAACqaXHeAAAAAXNSR0IArs4c6QAAA69JREFUeAHtWs1uEzEQtjeRKBISBw5wKD9vAVc4QFcgVA4hZyRoX6OvQUHiHOVABUKBC1zpYyCCgEMOIKQGKV0zs1qn3l27tnftrUO8UhV7djw/n2fGP11C4hMRiAisMwK0qfPD4TDNsmwfxm8qZEyTJNkZjUYTxXsjsm89iZEVEiaN8zhis+CRjDYn+dbTGAB00MANEx6dGBMZJjxSPW0AkApcNWKtBhjknNTH8XicyxoMBkzK4IjYQo+0JtUiwCDnHLnSuRhpTaoBAGY1zqfOXbJXWPNNBoC92BUe0dfZznOuymea66rxVXmqfls9uvExAlTI29J1SNvKU/G71rP2EdAGgKlqlgS6CY/ALm2ayDDhkQpvDAAedEDiaYrzjYdUqwXRt56+hS0l1uKUd7VE9NDxradxBHjw9UxERgDOBPaAlGprgOt1NyDfc1NiCoQ2I13bs/YRoK0BbU9zXc9oVZ+uhq19BEQAqiGzbn1tDWgLyN4n1j/88/cOOyYDSllKGFkkfbr77sHGB5SdvpmnJGP7cJVcu6+T6Yar5y/ieBmPDc0LAKLTn3/NH4FBl9AoVlyYZwv2HLo3kGbjPLKDiOul8Uhs8TgHAGf08Pf8FTh7WWUXOsHfQdto5jl/8dur9Bt33RdBxl6c5nxjS4WBjND3QrdV03kEgPOZxKIZ0PI0qL6DnJ7yKMD2ZPt86Y4B0wnS6Ic4nvbIuCqnad95BNCE7lJKvoFB3wmhL2mP3rt1ceOK0sCE7qDj+EegXeXDAgo0EbzZzQvnPlb5mvadR8Dk4QZ+D1DKayRsHRxJbSz4S7MuMtKMPIYIER76eu82XQiEVk3nEdDKmspgDP+MsW2R7DL8UW7QAPgO/+ABwM2TOPtQU5yGP8p2XgPKBpv35DvCcvbDNuhpenC0hcWyqB3mChSc4aSA4XY4XzKBV+GPNTkcAKxNdzMgHACK/YDOLdV+QTdO9T6YGiDuB9K387vsmOWnxcLwGW6mXK7/HJBwIoBbBL9dVH+uLjgAcPMDEOARevm43vwsBUMjOAC62PyIAHirAfJ1/UQ1nA3yRZ4XNb6udxn+aI2/CGi8rrP7JzDB3s/h0VeUy9v+AOAaLH/hKL28T4D2T5dHX5kp/gBouq5T+gwM/ZrfKVD6xMfSJwLhrQaI67qoUNcuxl3T8bl67y8CXFnoWU4EwDPAwYvX1gDdf1eD91BjYEwBCUBTCe1/IdV8q0WAwZeZqwqGky9XV9X5aHdEICIgR+AfGqgwglFLPQMAAAAASUVORK5CYII=</icon>
+    <properties>
+        <property key="dataSource" type="enumeration" defaultValue="context">
+            <caption>Data source</caption>
+            <category>Data source</category>
+            <description/>
+            <enumerationValues>
+                <enumerationValue key="context">Context</enumerationValue>
+                <enumerationValue key="XPath">Database</enumerationValue>
+                <enumerationValue key="microflow">Microflow</enumerationValue>
+                <enumerationValue key="nanoflow">Nanoflow</enumerationValue>
+            </enumerationValues>
+        </property>
+        <property key="eventEntity" type="entity" allowNonPersistableEntities="true">
+            <caption>Event entity</caption>
+            <category>Data source</category>
+            <description>With data context, this entity should be equal to the containing data view</description>
+        </property>
+        <property key="entityConstraint" type="entityConstraint" entityProperty="eventEntity" required="false">
+            <caption>XPath</caption>
+            <category>Data source</category>
+            <description/>
```

Comment:
Add description, "Also allows the event objects to be constraint to a relation to context by using `[%CurrentObject%]`

Reply:
Done